GitUI/CommandsDialogs/RepoHosting/CreatePullRequestForm.cs

```diff
@@ -191,5 +191,24 @@ private void _createBtn_Click(object sender, EventArgs e)
                     ex.Message, _strError.Text, MessageBoxButtons.OK, MessageBoxIcon.Error);
             }
         }
+
+        /// <summary>
+        /// Clean up any resources being used.
+        /// </summary>
+        /// <param name="disposing">true if managed resources should be disposed; otherwise, false.</param>
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing)
+            {
+                _remoteLoader.Cancel();
+                _remoteLoader.Dispose();
+            }
+
+            if (disposing && (components != null))
```

Comment:
:bulb: this block could be added to the previous block to simplify the condition.